components/OONIRunHero.js

```diff
@@ -8,15 +8,20 @@ import {
 } from 'ooni-components'
 
 import styled from 'styled-components'
+
 const BrandContainer = styled.div`
   max-width: 100%;
+  height: 95px;
+  margin-top: 20px;
+
   svg {
     max-width: 100%;
   }
 `
+
 export default props => {
   return (
-    <Hero pb={4} pt={4}>
+    <Hero pb={4} pt={4} sx={{maxWidth: '100% !important'}}>
```

Comment:
Why do we have to add `maxWidth: 100% !important`? Shouldn't this be set directly in the `Hero` component?

Reply:
Reason why I had to use `!important` here is because the design-system version present in **run** is [v0.2.10](https://github.com/ooni/design-system/tree/d0abef874ed2f7daa20219f9f882b61cacdeb03f) 
At that time, it was not having any kind of [variants](https://github.com/ooni/design-system/blob/d0abef874ed2f7daa20219f9f882b61cacdeb03f/components/theme/rebassTheme.js), but now we have.

The way hero component was developed was also different [at that time](https://github.com/ooni/design-system/blob/d0abef874ed2f7daa20219f9f882b61cacdeb03f/components/organisms/Hero.js). It was having `width: 100%` defined. But now, if we try to make any change in `Hero` component, it can affect the things in other projects using `Hero` component.

That's why in order to reduce breaking style change (by changing in base design-system), I had to add the changes here and for some reason setting without using `!important` was not even working out maybe because of the **variants**.

The same reason is for the `<Container />` component. 

I think the solution for this will be that either we keep it like this and then in the next major release of **design-system** we remove the **rebass** dependency. By doing that, we can define all new variants as per the projects. That'll also bring all the projects to same level with respect to the design-system dependency.

Or, we don't merge the current release changes to **ooni-run**. We start adding the changes after removing the rebass dependency. 🤔